From khali@linux-fr.org Mon Oct 17 13:58:47 2005
Date: Mon, 17 Oct 2005 22:58:35 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 05/14] Owner field additions to many i2c drivers, 5 of 5
Message-Id: <20051017225835.418f46f4.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-owner-field-05-i2c-isa.patch

From: Laurent Riffard <laurent.riffard@free.fr>

In function i2c_isa_add_driver, copied driver should inherit the owner
field as well as the name field.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-isa.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-isa.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-isa.c
@@ -92,6 +92,7 @@ int i2c_isa_add_driver(struct i2c_driver
 
 	/* Add the driver to the list of i2c drivers in the driver core */
 	driver->driver.name = driver->name;
+	driver->driver.owner = driver->owner;
 	driver->driver.bus = &i2c_bus_type;
 	driver->driver.probe = i2c_isa_device_probe;
 	driver->driver.remove = i2c_isa_device_remove;
